src/main/java/org/opentripplanner/graph_builder/module/islandpruning/PruneIslands.java

```diff
@@ -343,7 +344,8 @@ private void collectNeighbourVertices(
             e instanceof StreetEdge ||
             e instanceof ElevatorEdge ||
             e instanceof FreeEdge ||
-            e instanceof StreetTransitEntityLink
+            e instanceof StreetTransitEntityLink ||
+            e instanceof EscalatorEdge
```

Comment:
@vesameskanen Would it be a good idea to convert these `instanceOf` checks into a method on `Edge` instead?

(I'm not going to expect @nurmAV to do it in this PR.)

Reply:
I found out that helsinki area pruning works exactly the same way, when all edges are considered. I suggest removing edge class test. 

I am sorry that this comment comes this late. Somehow the new test now appears redundant.

Reply:
Good investigation. I think this code may have been from a time when we had transit edges (`TransitHopEdge`) in the graph and you would probably never want to prune them.